eng/pipelines/runtime.yml

```diff
@@ -136,6 +136,7 @@ jobs:
     - Windows_NT_x86
     - Windows_NT_arm
     - Windows_NT_arm64
+    - FreeBSD_x64
```

Comment:
Same as comment above, these steps are going away here: https://github.com/dotnet/runtime/pull/34664 and being merged into libraries build-job.